YAPA/MainWindow.xaml.cs

```diff
@@ -598,7 +586,9 @@ private void Stop_Click(object sender, RoutedEventArgs e)
             else
             {
                 ResetTicking();
+                _isWork = false;
                 PlayMusic();
+                _isWork = true;
```

Comment:
by the way, I set the ticktock sound for workMusic, set a Ding sound for the breakMusic. so when I'm working, I'm always listen to the ticktock for 25 minutes(that's why I need it playlooping), after 25 mins work time, it plays a simple ding sound, to let me know this promodo has finished, and I don't want to hear the ding sound over and over again, that's why I  don't loop it.

but now I think this _ding_ sound should be played by the sound notifications, not by breakMusic. but I disabled to sound notifications in setting page, the reason is, my workmusic is a endless ticktock, but the sound notifications when starting is another ticktock, so it's wired to put it together.

maybe my comments is not clear enough, hope you could understand.